### ISSUE TITLE
[BUG] `kubectl top pods` is not working

### ISSUE THREAD

**Issue by UnderGreen** [gravitational/gravity#866]
**Describe the bug**

`kubectl top pods` returns:
```
W1104 21:37:56.517807   15696 top_pod.go:266] Metrics not available for pod default/mattermost-database-5bgzw, age: 9m6.517800322s
error: Metrics not available for pod default/mattermost-database-5bgzw, age: 9m6.517800322s
```
**To Reproduce**

**Expected behavior**

**Logs**

**Environment (please complete the following information):**
 - OS [e.g. Redhat 7.4]:
 - Gravity [e.g. 5.5.4]: 6.2.x+ (6.1?)
 - Platform [e.g. Vmware, AWS]:

**Additional context**

I believe it is because of some changes in metrics names or tags from kubelet in K8s 1.14+ and [this line](https://github.com/gravitational/monitoring-app/blob/master/resources/prometheus/prometheus-adapter-configMap.yaml#L6) does not catch them correctly.

**Comment by yjjnls** [comment]
any idea？

**Comment by UnderGreen** [comment]
Hi @yjjnls. Do you experience the issue above in any of the latest gravity versions? I fixed the issue for 7.0.x, 6.1.x.
Which gravity version do you use?

**Comment by yjjnls** [comment]
@UnderGreen it fixed in higher version
